### DISCUSSION TITLE
How to save ocrized pdf in doctr?

### DISCUSSION THREAD

**Discussion by chatgptguru** [mindee/doctr#1497] [Q&A]
Is it possible to export ocrized pdf in doctr? 
Now, I have confirm that your ocr is working fine and am happy about your ocr result. Thank you so much. Btw, I need to save ocrized pdf file and use that. Please help. 

My code:
from doctr.models import ocr_predictor
from doctr.io import DocumentFile
doc = DocumentFile.from_pdf("test.pdf")
result = model(doc)

**Comment by felixdittrich92** [comment]
Hi @chatgptguru, glad to hear that you like docTR :hugs: you mean as PDF/A-File (PDF with text layer) ?

See: 
https://www.mindee.com/blog/create-ocrized-pdfs-in-2-steps

and:
https://github.com/mindee/notebooks/blob/main/doctr/export_as_pdfa.ipynb
